WalletWasabi/Crypto/Mac.cs

```diff
@@ -17,15 +17,13 @@ private MAC(Scalar t, GroupElement v)
 		public GroupElement V { get; }
 		public GroupElement U => GenerateU(T);
 
-		public static bool operator ==(MAC a, MAC b) => a.Equals(b);
```

Comment:
@lontivero Please take a note here. Originally if `a` was `null`, it failed, but if `b` was `null`, it didn't fail.